5. Asynchronous/03. Serial Promises.md

```diff
@@ -22,4 +22,21 @@ async function parallelApiCalls() {
     const [firstResult, secondResult] = await Promise.all([fetchFirst(), fetchSecond()]);
     return [firstResult, secondResult];
 }
-```
\ No newline at end of file
+```
+
+## 3️⃣ Asynchronous loop serial calls
+
+Se ejecuta de forma secuencial un array de promesas.
```

Comment:
Please, use english in this description. I have translated the other texts in the file